chapters/VK_EXT_present_timing/PresentTimeInfo.adoc

```diff
@@ -0,0 +1,162 @@
+    image for any duration allowed by the current present mode.
+
+The contents of the sname:VkPresentTimeEXT union is interpreted depending on
+the associated sname:VkPresentInfoKHR::pname:pSwapchains member creation
+flags. If ename:VK_SWAPCHAIN_CREATE_ABSOLUTE_TIME_BIT_EXT was used to create
+the pname:swapchain, pname:targetPresentTime is used. If
+ename:VK_SWAPCHAIN_CREATE_RELATIVE_TIME_BIT_EXT was used to create the
+pname:swapchain, pname:minPresentDuration is used.
+
+pname:targetPresentTime is a time in nanoseconds, according to the
+time-domain being used. The presentation engine must: not complete the
+target present stage, specified in the associated swapchain
+slink:VkSwapchainPresentTimingCreateInfoEXT::pname:presentStageTarget, at an
+earlier time.
+
+[NOTE]
+.Note
+====
+Applications that use pname:targetPresentTime to specify an absolute present
+time should: regularly rebase their calculations for their next target time
+on the feedback from flink:vkGetPastPresentationTimingEXT to compensate for
+accumulated precision errors or potential clock drift. It is recommended
+that when targeting the time of a vertical blanking period, applications
+subtract a small delta from pname:targetPresentTime as a margin for error.
+====
+
+include::{generated}/validity/structs/VkPresentTimeEXT.adoc[]
+--
+
+[open,refpage='VkPresentStageTimeEXT',desc='Associate a present stage with a timestamp',type='structs']
+--
+The sname:VkPresentStageTimeEXT structure is defined as:
+
+include::{generated}/api/structs/VkPresentStageTimeEXT.adoc[]
+
+  * pname:stage is a tlink:VkPresentStageFlagsEXT value specifying a present stage.
+  * pname:time is a time in nanoseconds associated with the pname:stage.
+
+include::{generated}/validity/structs/VkPresentStageTimeEXT.adoc[]
+--
+
+[open,refpage='VkPresentStageFlagBitsEXT',desc='Bitmask specifying stages of the image presentation process',type='enums']
+--
+
+Presenting an image to the user typically involves multiple stages. Bits
+which can: be set to specify present stages are:
+
+include::{generated}/api/enums/VkPresentStageFlagBitsEXT.adoc[]
+
+  * ename:VK_PRESENT_STAGE_IMAGE_HANDOFF_BIT_EXT is the stage after which the
+    presentation engine has passed the presentation request of the image to
+    a system-specific entity that is outside the control of the Vulkan
+    implementation. This stage is the only ename:VkPresentStageFlagBitsEXT
+    that is required: to be supported.
+  * ename:VK_PRESENT_STAGE_IMAGE_LATCHED_BIT_EXT is the stage after which the
+    image has been latched by a downstream entity, such as a compositor, and
+    scheduled to be presented to the user.
+  * ename:VK_PRESENT_STAGE_IMAGE_FIRST_PIXEL_OUT_BIT_EXT is the stage after
+    which data for the first pixel of the presentation request associated with
+    the image has left the presentation engine or another downstream entity for
+    a display hardware.
```

Comment:
Not sure how to make this more accurate but again if the PE is the whole system then we need some other thing here.